pkg/cronjob/es_index_cleaner_test.go

```diff
@@ -39,6 +40,38 @@ func TestCreateEsIndexCleaner(t *testing.T) {
 	assert.Equal(t, historyLimits, *cronJob.Spec.SuccessfulJobsHistoryLimit)
 }
 
+func TestCreateEsIndexCleanerTypeMeta(t *testing.T) {
+	testData := []struct {
+		Name string
+		flag string
```

Comment:
What's the reason behind Name being an exported variable and flag is unexported?